packages/devtools/src/launcher.js

```diff
@@ -109,7 +120,8 @@ function launchBrowser (capabilities, product) {
             width: DEFAULT_WIDTH,
             height: DEFAULT_HEIGHT
         }
-    }, capabilities[vendorCapKey] || {})
+    }, capabilities[vendorCapKey],
+    ignoreDefaultArgs ? { ignoreDefaultArgs: true } : {})
```

Comment:
same as above:

```js
    const puppeteerOptions = Object.assign({
        product,
        executablePath,
        defaultViewport: {
            width: DEFAULT_WIDTH,
            height: DEFAULT_HEIGHT
        },
		ignoreDefaultArgs: Boolean(ignoreDefaultArgs)
    }, capabilities[vendorCapKey] || {})
```

Reply:
> Could you move `ignoreDefaultArgs` into `goog:chromeOptions` as it i a custom capability?

It's true my first need was for Chrome. But:
* As I can see in [devtools/launcher.js](https://github.com/webdriverio/webdriverio/blob/master/packages/devtools/src/launcher.js#L129) devtools _can_ also be used for other browsers than Chrome. Maybe not yet at the same level as with Chrome, but it's possible.

* And it seems Puppeteer accepts this `ignoreDefaultArgs` parameter in its experimental support of Firefox (see [experimental/puppeteer-firefox/Launcher.js](https://github.com/puppeteer/puppeteer/blob/v3.0.2/experimental/puppeteer-firefox/lib/Launcher.js#L88)).

so we limit this parameter only for Chrome?

Reply:
> same as above:
> 
> ```js
>     const puppeteerOptions = Object.assign({
>         product,
>         executablePath,
>         defaultViewport: {
>             width: DEFAULT_WIDTH,
>             height: DEFAULT_HEIGHT
>         },
> 		ignoreDefaultArgs: Boolean(ignoreDefaultArgs)
>     }, capabilities[vendorCapKey] || {})
> ```

line 124 is actually wrong. It should be like line 75:
```js
    ignoreDefaultArgs ? { ignoreDefaultArgs: ignoreDefaultArgs } : {}
```
as `ignoreDefaultArgs` is either a _boolean_ or a _array of string_ (as in [Puppeteer](https://pptr.dev/#?product=Puppeteer&version=v3.0.2&show=api-puppeteerlaunchoptions))